### ISSUE TITLE
Ticketing Platform is not Accessible

### ISSUE THREAD

**Issue by bravegnu** [pythonindia/inpycon2019-tasks#90]
The ticketing platform is not accessible, and I don't think this is going to be fixed any time soon. We need to have an alternative mechanism in place, for people who use screen readers.

One option is provide a email ID, where they can mail us their details, and we can get in touch with them, and ask them wire the amount to our bank account. And then we will need to generate tickets from the ticketing platform.

If there are better ideas let me know.

**Comment by sharmi** [comment]
Paypal is accessible. https://www.afb.org/blindness-and-low-vision/using-technology/online-shopping-and-banking-accessibility-people-visual-2

Wondering if we can use paypal.me or similar feature . You could also send a request for payment link over email. They can click on the link and pay.

https://www.paypal.com/lu/smarthelp/article/how-do-i-request-payments-or-send-an-invoice-via-paypal-faq2155

**Comment by bravegnu** [comment]
@sharmi Does that require sign-up / setup from the user side? If yes, that might add to the burden.

Given that there will be only few users using this mode, we can deal with it using NEFT. Anybody who already has a PayPal account can transfer using PayPal NEFT.

**Comment by sharmi** [comment]
Yes, sign-up will be required.  So NEFT will be better.

Hopefully, they would have opted for an account in a bank that is has
accessible website.

**Comment by bravegnu** [comment]
We will need a few volunteers who can manage these requests. @sharmi Can you arrange for that from the Diversity workgroup?

**Comment by abhishekmishragithub** [comment]
> The ticketing platform is not accessible, and I don't think this is going to be fixed any time soon. We need to have an alternative mechanism in place, for people who use screen readers.
> 
> One option is provide a email ID, where they can mail us their details, and we can get in touch with them, and ask them wire the amount to our bank account. And then we will need to generate tickets from the ticketing platform.
> 
> If there are better ideas let me know.

@bravegnu can't we use 3rd party payment gateways like instamojo or eventbrite ?

**Comment by bravegnu** [comment]
@abhishekmishragithub We are already partnering with Townscript, for ticketing, which I ran accessibility tests on. I am not sure about the other platforms, I assume they also have similar problems. Can you check?

**Comment by bravegnu** [comment]
My understanding is that almost all ticketing platforms are broken one way or the other, and fixing them up, at this point is not going to work. It is better to have a work around.

**Comment by bravegnu** [comment]
Spoke to Townscript regarding this, they said they will discuss internally and get back to us, by tomorrow, on how they can improve the accessibility of their platform. But as a stopgap we might have a mail + NEFT based approach to book tickets.

**Comment by sharmi** [comment]
Do we have an official mail id Vijay? One with @in.python.org? Since this
is regarding payment, emails from personal mail ids might not look very
trustworthy/professional.

**Comment by bravegnu** [comment]
Will be setting up one to receive requests: tickets@in.pycon.org

**Comment by abhishekmishragithub** [comment]
@bravegnu so did we finalize method for ticket? Ticket platform is now accessible ?

**Comment by bravegnu** [comment]
No, we are planning for a workaround.

**Comment by ParitoshikPaul** [comment]
@bravegnu: Have a look at instamojo https://www.instamojo.com/
I have tried this for one of my events, Its light weight / Do not require any signup and provides a link to specific tickets.
I have seen townscript but it takes you to a page of tickets which might be a duplicate step of what we have on our website.
Please check and let me know, Also there is no integration required its just a link to pay.

**Comment by bravegnu** [comment]
Thanks @ParitoshikPaul for the inputs. I don't think we would want to switch the ticketing platform at this point in time though.

**Comment by ParitoshikPaul** [comment]
Yeah its high time as the tickets are out, we can keep it as a future reference.
